src/pages/home/dialogs/NewRecordsDialog.vue

```
@@ -0,0 +1,86 @@
+<script setup>
```

ESLint

[error] The 'lang' attribute of '<script>' is missing.

(vue/block-lang)